docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,429 @@ aliases:
 
 # Deploying and managing Brigade
 
-TODO
\ No newline at end of file
+In this doc, we'll go over steps for setting up and managing a full,
```

Comment:
Total nit: I'd strike "full" just because I don't think it adds value on top of the "production-grade".